deploy/nginx.service.yml.tpl

```diff
@@ -1,23 +1,22 @@
-# This is the backend service
-apiVersion: v1
 kind: Service
+apiVersion: v1
```

Comment:
Is there a reason for switching the kind and apiVersion positions?

Reply:
no, it does not matter. I have copied this file from kubernetes.